homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -608,16 +608,27 @@ async def async_media_next_track(self) -> None:
 
     async def async_media_seek(self, position: float) -> None:
         """Seek to position in ms."""
-        if self._source_change.id == BangOlufsenSource.DEEZER.id:
+        if self._source_change.is_seekable:
```

Comment:
If the source is not seekable, the supported features should omit that instead of raising an error